scalar.h exists in a different directory than expected (this might
be a regression from a previous refactoring).

Should be offered upstream.

--- valgrind-3.15.0/memcheck/tests/x86-solaris/scalar.c
+++ valgrind-3.15.0/memcheck/tests/x86-solaris/scalar.c
@@ -1,6 +1,6 @@
 /* Basic syscall test for Solaris/x86 specific syscalls. */
 
-#include "scalar.h"
+#include "../solaris/scalar.h"
 
 #include <string.h>
 #include <sys/fcntl.h>
--- valgrind-3.15.0/memcheck/tests/amd64-solaris/scalar.c
+++ valgrind-3.15.0/memcheck/tests/amd64-solaris/scalar.c
@@ -1,6 +1,6 @@
 /* Basic syscall test for Solaris/amd64 specific syscalls. */
 
-#include "scalar.h"
+#include "../solaris/scalar.h"
 
 #include <sys/lwp.h>
 
